examples/tensorflow/large_language_model/README.md

```diff
@@ -0,0 +1,24 @@
+1. Get fp32 accuracy
```

Comment:
The example structure is like `examples/[framework]/[model domain]/[model name]/[optimization method]/...`, so please refactor the example path for LLM models.

Reply:
OK, got it